### PR TITLE
jailhouse: cleanup and fix bitbake tasks overrides

### PR DESCRIPTION
**Pull request by quaresmajose**
No functional changes but is need to works as expected on main-next

### CODE REVIEW

**Comment by igoropaniuk** [comment]
One type in the commit message:
> The native python is only needed by the setuptools **witch** is
already provided in setuptools3

File: meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb [hunk 1]
```diff
@@ -47,6 +47,17 @@ INMATES_DTB_DIR ?= "${JH_DATADIR}/inmates/dtb"
...
+EXTRA_OEMAKE += 'V=1'
+EXTRA_OEMAKE += 'PYTHON=python3'
+EXTRA_OEMAKE += 'LDFLAGS=""'
+EXTRA_OEMAKE += 'CC="${CC}"'
+EXTRA_OEMAKE += 'ARCH=${JH_ARCH}'
+EXTRA_OEMAKE += 'CROSS_COMPILE=${TARGET_PREFIX}'
+EXTRA_OEMAKE += 'KDIR=${STAGING_KERNEL_BUILDDIR}'
+EXTRA_OEMAKE += 'MODLIB="${D}${nonarch_base_libdir}/modules/${KERNEL_VERSION}"'
+EXTRA_OEMAKE += 'INSTALL_MOD_PATH=${D}${root_prefix}'
+EXTRA_OEMAKE += 'firmwaredir=${nonarch_base_libdir}/firmware'
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb:59]: No need for multiple lines, one with multiple lines terminating with \ should be enough.

File: meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb [hunk 2]
```diff
@@ -47,6 +47,17 @@ INMATES_DTB_DIR ?= "${JH_DATADIR}/inmates/dtb"
...
 JH_INMATE_DTB ?= ""
 
+EXTRA_OEMAKE += 'V=1'
+EXTRA_OEMAKE += 'PYTHON=python3'
+EXTRA_OEMAKE += 'LDFLAGS=""'
+EXTRA_OEMAKE += 'CC="${CC}"'
+EXTRA_OEMAKE += 'ARCH=${JH_ARCH}'
+EXTRA_OEMAKE += 'CROSS_COMPILE=${TARGET_PREFIX}'
+EXTRA_OEMAKE += 'KDIR=${STAGING_KERNEL_BUILDDIR}'
+EXTRA_OEMAKE += 'MODLIB="${D}${nonarch_base_libdir}/modules/${KERNEL_VERSION}"'
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb:57]: This is an extra parameter, please add to the commit message why it was added.

Comment (quaresmajose) [reply meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb:57]: are you asking explicitly about the MODLIB parameter?

File: meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb (hunk 1 above)
Comment (quaresmajose) [reply meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb]: fixed

File: meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb (hunk 2 above)
Comment (ricardosalveti) [reply meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb]: Yes

**Comment by igoropaniuk** [comment]
with @ricardosalveti comment addressed LGTM

**Review by MrCry0 — APPROVED** [review]
LGTM with Ricardos's notes

**Comment by quaresmajose** [comment]
jailhouse will be dropped in https://github.com/foundriesio/meta-lmp/pull/1515

**Pull request closed without merging**